### ISSUE TITLE
Error management

### ISSUE THREAD

**Issue by gigit-f** [muquit/mailsend-go#42]
Hi to all.
How can I manage errors when mailsend-go is launched via a batch file or within a program?
Is there a return code or a list of possible errors? 
Many thanks.

**Comment by keerah** [comment]
yes, would love to have this feature, or at least to have an option for mailsend-go to retry sending for some time or/and retry attempts number
